app/controllers/assessments_controller.rb

```diff
@@ -101,6 +126,8 @@ def create
       redirect_to(root_url) && return unless patient
 
       threshold_condition_hash = params.permit(:threshold_hash)[:threshold_hash]
+      redirect_to(root_url) && return if threshold_condition_hash.blank?
```

Comment:
@holmesie I added this redirect in the `create` because I found a bug in this PR where I could add assessments that would show up fine in the table, but actually had ReportedConditions with a nil threshold_condition_hash because for some reason, there is always a single ThresholdCondition with a nil hash in every environment so the redirect below this one was not being hit. Do you know why that single ThresholdCondition with a nil hash seems to exist?

Reply:
Update after speaking with @mayerm94. There are more than one instance of ThresholdConditions with `nil` hashes, and these are the threshold conditions at the individual jurisdiction level. The `ThresholdConditions` with a hash represents the condition that is the sum of all the threshold conditions (without hashes) throughout the jurisdiction path === state 1s custom symptoms + all of USAs symptoms, so the ones with `nil` hashes will be aggregated up. No assessments should point to a threshold condition with a nil hash, so this redirect should be safe.